packages/terra-button-group/src/terra-dev-site/test/button-group/ButtonGroupIcon.test.jsx

```diff
@@ -1,11 +1,7 @@
 import React from 'react';
-import classNames from 'classnames/bind';
 import ButtonGroup from '../../../ButtonGroup';
-import styles from './ButtonGroupIcon.test.scss';
```

Comment:
If we no longer need this .scss file, we can delete it.

Reply:
Deleted in 2ff8a50

Reply:
Resolved in 2ff8a50